bodhi/server/models.py

```diff
@@ -789,6 +789,14 @@ class Update(Base):
     user_id = Column(Integer, ForeignKey('users.id'))
 
     @property
+    def mandatory_days_in_testing(self):
+        if self.critpath:
+            return int(config.get('critpath.stable_after_days_without_negative_karma'))
+
+        days = self.release.mandatory_days_in_testing
+        return days if days else 0
```

Comment:
We should add three tests for this new method:

1. Test a critpath update to make sure this returns an integer of the setting.
2. Test a regular update with ```release.mandatory_days_in_testing``` set to some "truthy" value, like 7.
3. Test a regular update with ```release.mandatory_days_in_testing``` set to some "falsy" value, like None.